src/App.js

```diff
@@ -17,7 +17,30 @@ const styles = theme => ({
     paddingTop: theme.spacing.unit * 20,
   },
 });
-
+class Tile extends React.Component{
+  render(){
+    const tileStyle = {
```

Comment:
Make a separate `Tiles.scss` file since we're using SASS as our CSS pre-processor.
Move these styles into that file and use classes to go with that.

Reply:
where should i put Tiles.scss file in Tiles folder or in assets/sass/ folder ?

Reply:
In the `Tiles.scss` folder